src/olympia/amo/tests/test_commands.py

```diff
@@ -61,7 +65,7 @@ def test_compress_assets_command_without_git():
 
     # Call command a second time. We should get a different build id, since it
     # depends on a uuid.
-    call_command('compress_assets')
+    call_command('compress_assets', stdout=out)
```

Comment:
I'm guessing there's no way to do `stdout=None` to avoid a needless variable being created which we then don't do anything with.  Meh.

Reply:
No, unfortunately not. Django does `stdout or sys.stdout` so `None` would have the side effect of `sys.stdout` being used.